hub/core/transform/transform.py

```diff
@@ -25,48 +27,58 @@
 
 def transform(
     data_in,
-    pipeline: Sequence[Callable],
+    pipeline: Union[Callable, Sequence[Callable]],
     ds_out: Dataset,
-    pipeline_kwargs: Optional[Sequence[Dict]] = None,
+    pipeline_kwargs: Optional[Union[Dict, Sequence[Dict]]] = None,
     scheduler: str = "threaded",
     workers: int = 1,
 ):
 
     """Transforms the data_in to produce an output dataset ds_out using one or more workers.
     Useful for generating new datasets or converting datasets from one format to another efficiently.
     Eg.
```

Comment:
Can you follow the standard format for examples?
```
Examples:
    >>> transform(...)
    ...
```
And add some line breaks between sections? This massive text block is really hard to navigate at a glance, and this is what will be used by the api reference.